lhotse/dataset/speech_recognition.py

```diff
@@ -14,7 +14,16 @@
 class SpeechRecognitionDataset(Dataset):
     """
     The PyTorch Dataset for the speech recognition task.
-    Contains acoustic features and the corresponding text.
+    Returns a dict of:
```

Comment:
When naming this, we might want to bear in mind that in future there may be other versions of this.
E.g. we may want to do padding while collating, to ensure even minibatch sizes.  
(I'm thinking that may be easier to do by implementing the data-loader with no underlying Dataset,
just reading the CutSet directly).
Up to you though.

Reply:
Yeah, there'll definitely be more versions; we should rename as necessary. Re collating, check out my previous comment.